### ISSUE TITLE
Opacity incompatibility with Dark Reader add-on in Firefox.

### ISSUE THREAD

**Issue by Kratorator** [luben/load-progress-bar#19]
Even with Opacity set at maximum, it is still (maybe 50%?) transparent when used with Dark Reader, in both Dark and Light mode. I also posted an issue in https://github.com/darkreader/darkreader.

Maybe you two could communicate about this (small) problem.

**Comment by luben** [comment]
Yes, the min opacity is 25%: https://github.com/luben/load-progress-bar/blob/master/settings/options.html#L39 - this does not depend on darkreader or not.

Can I ask why you want opacity to be lower?

**Comment by Kratorator** [comment]
Hi,

I want FULL opacity when it's set at full opacity! (no transparency)

Bye,

chrr

**Comment by luben** [comment]
Yes, but why? If we allow full opacity the progress bar will not show at all. This will lead to users complaining that the add-on does not work.

**Comment by Kratorator** [comment]
JHCOAFRC

It seems that you confuse opacity with transparency.

When I drag the slider for Thickness all the way to the right, it is at 
100% thickness, right? so when I drag the slider for Opacity all the way 
to the right, it's at 100% opacity, right?

Opacity is the opposite of transparency.
	100% OPACITY = 0% TRANSPARENCY = FULLY VISIBLE.
	100% TRANSPARENCY = 0% OPACITY = invisible.

I want the Load Progress Bar to be FULLY VISIBLE, so I drag the slider 
to 100% OPACITY. It works fine WITHOUT Dark Reader.

My progress bar color is RGB(255, 255, 0) which is fluo yellow. Very 
visible. If I turn ON Dark Reader, that yellow is NOT FULLY OPAQUE (=not 
100% opacity), even though the opacity slider IS at 100% opacity, all 
the way to the right. The progress bar is the color of sick old mustard, 
because it is partly opaque (maybe 60%) and partly transparent (maybe 
40%). This only happens when Dark Reader is active.

Ciao,

chrr

**Comment by luben** [comment]
Hmm, yes, I was confusing the opacity/transparency.  Looked at the code - it sets it to 100%. Tried it in combination with the darkreader extension and indeed the color looks muted. On closer inspection, the dark-reader installs a global style and I guess that  also affects the color of the progress bar. I still cannot figure how it does that and is it possible to work around it.
